apps/src/dance/ai/utils/chooseEffects.ts

```diff
@@ -1,19 +1,29 @@
 import {cachedWeightsMappings} from '../constants';
-import {FieldKey, GeneratedEffect, EffectsQuality} from '../types';
-
+import {
+  FieldKey,
+  GeneratedEffect,
+  EffectsQuality,
+  CachedWeightsMapping,
+} from '../types';
 import {calculateOutputSummedWeights} from './calculateOutputSummedWeights';
-import {getSortedOptions} from './getSortedOptions';
 
 const NUM_RANDOM_TOP_OPTIONS = 3;
 const NUM_RANDOM_BOTTOM_OPTIONS = 20;
 
+type CalculateSummedWeights = (
```

Comment:
Is there a reason you didn't put this type in dance/ai/types?

Reply:
It's only used here so I thought I'd keep it here for simplicity? Do we want all types in the types file?

Reply:
Not sure! Is that our heuristic if that the type is only used in one file, we keep it defined there? SGTM!

Reply:
^ that's generally been my approach, but not a strongly held opinion.